### DISCUSSION TITLE
Update does not update

### DISCUSSION THREAD

**Discussion by theontho** [standardnotes/forum#2469] [General]
**Describe the bug**
When I click install & restart with an update, app closes.  But when I open the app again it's still at the old version.

**To Reproduce**
1. Hamburger menu top left
2. (1) Update Available
3. Install Pending Update
4. Click Install and restart from alert box
5. App does not start again, no installer shows up
6. Open app manually
7. See still at old version

**Expected behavior**
App updates

**Screenshots**
<img width="356" alt="Screenshot 2020-11-25 165444" src="https://user-images.githubusercontent.com/22130/100295692-0fd51500-2f3f-11eb-8244-7d14933059d4.png">

**Desktop (please complete the following information):**
 - OS: Windows 10 App
 - Version 3.5.6

**Additional context**
The third time the installer finally showed up and the app updated

**Comment by JaspalSuri** [comment]
Hi @theontho,

I'm sorry to hear that; I'm glad that it was finally able to update to the latest version of the app. 

For anyone else reading this that doesn't want to try to install the app multiple times, you download the update directly [from here](https://github.com/standardnotes/desktop/releases/tag/v3.5.10) instead. You might have to click on the button near the word **Assets** to reveal the option(s) available for your platform.

@baptiste-grob I was able to reproduce this on macOS Big Sur (worked on the second install), but not on Windows 10. Also, is it possible for the app to reuse an existing installation package instead of having to download it again? (It might not be the best practice, though.)

**Comment by upward4** [comment]
I too have this issue, on Linux. I have to manually download the AppImage from GitHub in order to update. Could you please try to reproduce on a Linux machine?

**Comment by JaspalSuri** [comment]
I was able to reproduce it on Ubuntu, but only for certain versions of the app. I'll test it once more, though! Which Linux [distro](https://distrowatch.com/) are you using?

**Comment by upward4** [comment]
> I was able to reproduce it on Ubuntu, but only for certain versions of the app. I'll test it once more, though! Which Linux [distro](https://distrowatch.com/) are you using?

I am currently using Fedora 32. The issue is reproducible with Standard Notes v3.5.11.

**Comment by upward4** [comment]
The issue still persists when trying to update from 3.5.14 to 3.5.16.

Update: I resolved by updating AppImageLauncher to the latest version. Apparently it had a bug that caused updates to fail.

**Comment by JaspalSuri** [comment]
Sorry for the late reply. Thanks for letting us know! Our desktop dev has noticed that when the AppImage file is renamed, the app fails to update. They're working on a fix for this. 🙂
